NEXT_CHANGELOG.md

```diff
@@ -76,24 +76,27 @@ use apollo_router::ApolloRouter;
 use apollo_router::Configuration;
 use apollo_router::ConfigurationKind;
 use apollo_router::Context;
+use apollo_router::Error;
```

Comment:
Shouldn't this have been moved to `graphql::Error`? Likewise for the NewErrorBuilder, Response and Request.